File: app/src/main/java/com/example/moviescodechallenge/view/TVShowsView.java
package com.example.moviescodechallenge.view;

import com.example.moviescodechallenge.dto.Result;

/**
 * Created by fbarbieri on 2019-08-24.
 */
public interface TVShowsView {

    void showTVShowsData(Result tvShows);

    void showProgressBar();

    void hideProgressBar();
}

File: app/src/main/java/com/example/moviescodechallenge/view/TVShowsView.kt
package com.example.moviescodechallenge.view

import com.example.moviescodechallenge.dto.Result

/**
 * Created by fbarbieri on 2019-08-24.
 */
interface TVShowsView {

    fun showTVShowsData(tvShows: Result)

    fun showProgressBar()

    fun hideProgressBar()
}
